modules/recon/contacts-contacts/abc.py

```diff
@@ -0,0 +1,88 @@
+from recon.core.module import BaseModule
+
+from recon.mixins.threads import ThreadingMixin
+
+from bs4 import BeautifulSoup
+
+
+class Module(BaseModule, ThreadingMixin):
+
+    meta = {
+        "name": "Advance Background Check Lookup",
+        "author": "Cam Barts (@cam-barts)",
+        "version": "1.0",
+        "description": "Checks names at advancedbackgroundchecks.com",
```

Comment:
Punctuate, then update index.

Reply:
Do you mean add a period to the end of the description?

Reply:
Yes sir.